src/learning-path/learnig-path.ts

```diff
@@ -0,0 +1,15 @@
+import { HttpClientProxy } from '../utils/http-client-proxy';
+import { LearningPathDto } from './dto/learning-path.dto';
+
+export class LearningPath {
+  private readonly baseUrl: string;
+
+  constructor(host: string) {
+    this.baseUrl = `http://edtech-learning-path-runner.${host}`;
```

Comment:
@nataszczypiora it's `http://edtech-learning-path-runner-service.`

Reply:
changed